concent_api/delete-unsupported-messages.py

```diff
@@ -0,0 +1,40 @@
+import django
```

Comment:
Django should be imported after std lib imports. Generally order should be:

1. std lib imports.
2. 3rd party imports.
3. Django imports.
4. application specific imports.

Reply:
Done